corehq/messaging/smsbackends/telerivet/views.py

```diff
@@ -60,14 +60,21 @@ def message_status(request, message_id):
     status = request.POST.get('status')
     logger.info(f'Updating Telerivet message status: message_id={message_id}, status={status}')
 
-    process_message_status.delay(
-        message_id,
-        status,
-        request.POST.get('secret', ''),
-        error_message=request.POST.get('error_message', ''),
-    )
+    backend = SQLTelerivetBackend.by_webhook_secret(request.POST.get('secret', ''))
+    if backend is None:
+        return HttpResponseForbidden()
 
-    return HttpResponse()
+    try:
+        sms = SMS.objects.get(couch_id=message_id)
+        process_message_status(
+            sms,
+            status,
+            error_message=request.POST.get('error_message', ''),
```

Comment:
Is this passed in the request, `request.POST.get('error_message', '')` from the gateway?

Reply:
Jip

Reply:
you meant yup?

Reply:
O yeah...I guess it's only South African's then who does the "jip" slang (means the same) :)